content/learn/node-request-data.md

```diff
@@ -23,26 +21,23 @@ axios.post('https://whatever.com/todos', {
 This is the matching server-side code:
 
 ```js
-const express = require('express');
-
-const app = express();
-
-app.use(
-  express.urlencoded({
-    extended: true,
-  })
-);
-
-app.use(express.json());
+const http = require('node:http');
 
-app.post('/todos', (req, res) => {
-  console.log(req.body.todo);
+const server = http.createServer((req, res) => {
+  let data = '';
+  req.on('data', chunk => {
+    data += chunk;
+  });
+  req.on('end', () => {
+    console.log(JSON.parse(data).todo); // 'Buy the milk'
```

Comment:
Hmm, this could be confusing if you don't comment somewhere about what was the original message. And from where the `.todo` property came from.